Sources/OktaAnalytics/OktaAnalytics.swift

```diff
@@ -24,6 +24,7 @@ public final class OktaAnalytics: NSObject {
 
     private static var providers = [String: AnalyticsProviderProtocol]()
     private static var lock = ReadWriteLock()
+    public static let oneDayInSeconds: UInt = 24 * 60 * 60
```

Comment:
We need to give flexibility to set value from client side

Reply:
Below I added the function `getScenarioEventsCreatedBy(_ secondsAgo: UInt = oneDayInSecond, ...` and we can set `secondsAgo` as a parameter of this function, by default this value 24 hours and could be easy changed from the client side.